src/NServiceBus.Persistence.AzureStorage.ComponentTests/NServiceBus.Persistence.AzureStorage.ComponentTests.csproj

```diff
@@ -2,30 +2,23 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <TargetFramework>net452</TargetFramework>
-    <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
     <SignAssembly>true</SignAssembly>
     <AssemblyOriginatorKeyFile>$(SolutionDir)NServiceBus.snk</AssemblyOriginatorKeyFile>
     <DebugType>full</DebugType>
     <DebugSymbols>True</DebugSymbols>
-    <WarningLevel>4</WarningLevel>
-    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
     <NoWarn>PCR0001</NoWarn>
```

Comment:
@SimonCropp do you know what this `PCR0001` warning is? I find that all warnings we had were incorrect and were hiding things we should have done. Such as missing xmldoc on public members.

Reply:
that is because Particular.CodeRules enforces `ConfigureAwait` even when it is redundant, eg in a unit test project.  TBH i dont think Particular.CodeRules adds enough value and would remove it. but that is beyond the scope of this PR

Reply:
We're not using Particular.CodeRules in tests... so I'd 👍 removal within this PR. Up to you if you want.
Thank you for the explanation. Funny, RPC is abbreviated Particular.CodeRules. New things everyday.